Formula/osquery.rb

```diff
@@ -8,7 +8,7 @@ class Osquery < Formula
 
   bottle do
     cellar :any
-    revision 1
+    revision 2
```

Comment:
The bottle revision is different from the formula revision; the latter goes outside the bottle block just below the `sha256`.  But it seems like it's not necessary here since the build is passing as-is.